docs/tutorials/add-a-pallet/configure-a-pallet.md

```diff
@@ -30,42 +30,40 @@ trait for the Nicks pallet.
 
 To figure out what you need to implement for the Nicks pallet specifically, you can take a look at
 the
-[`pallet_nicks::Config` documentation](https://substrate.dev/rustdocs/v3.0.0-monthly-2021-05/pallet_nicks/trait.Config.html)
+[`pallet_nicks::Config` documentation](https://substrate.dev/rustdocs/latest/pallet_nicks/trait.Config.html)
```

Comment:
The use of latest and master may lead to conflicts in what the docs here have and what the source does. I would want to explicitly use a version here and make that known. This is vest practice for all of devhub I think too. What do you think?

Reply:
The use of `latest` tag is going to be different from master branch. `master` branch is more like a development branch in our case where we have features and fixes applied but not yet released.

`latest` tag is always referring to the latest RELEASED VERSION. And in our doc, we can then refer to this latest tagged version. From then on, we don't need to update the release number in the doc every time when a monthly release is made. And when a monthly release is made, we can just move the latest tag up to the latest monthly version.

Reply:
Ok now I understand where you are coming from, Although we need to scan for breaking changes in any case, and there may be deprecated docs that are missed just because people didn't realize moving the latest tag/branch should have docs changes. It's best practice to _never_ change a tag from what I hear, as it's assumed static. If people ask about something, we need to have them clarify _when_ they looked at the docs, not what docs they looked at (or ask for the commit)

Reply:
The rule about tag to not move is generally correct. I think the exception is on the `latest` tag (or the similar setting). The name from it implies it is going to be floating and like a symlink in an OS, pointing to the exact version.